### PR TITLE
[KO] Translate 'ko.json', 'validation.json', 'validation-inline.json' 

### PR DESCRIPTION
**Pull request by SilNex**

### CODE REVIEW

File: json/ko.json [hunk 1]
```diff
@@ -247,363 +247,363 @@
...
-    "Please provide your name.": "Please provide your name.",
-    "Poland": "Poland",
-    "Portugal": "Portugal",
-    "Press \/ to search": "Press \/ to search",
-    "Preview": "Preview",
-    "Previous": "Previous",
+    "Please provide your name.": "이름을 입력하세요.",
+    "Poland": "폴란드",
+    "Portugal": "포르투갈",
+    "Press / to search": "/을 눌러 검색",
```
Comment (caouecs) [inline json/ko.json:403]: don't change the tag please, and I think \\/ is needed in translation

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you

just a problem in json file

**Comment by caouecs** [comment]
with the v9 of this package, the files are been moved

**Comment by andrey-helldar** [comment]
@SilNex, we have changed the structure of the project.

Files has been moved to a new location:

| old | new |
|:---|:---|
| `json/ko.json` | `locales/ko/ko.json`|
| `src/ko/validation-inline.php` | `locales/ko/validation-inline.php`|
| `src/ko/validation.php` | `locales/ko/validation.php`|

**Comment by andrey-helldar** [comment]
Run the commands:

```bash
git fetch --all
git checkout -b "patch/d58d7a0" "d58d7a08d64e78a4a3d2f7bfa9eefd58831b19ed"
git switch master
git rebase patch/d58d7a0 master
git push
```

**Comment by caouecs** [comment]
thank you

**Pull request merged**